exercise_02.py

```diff
@@ -0,0 +1,25 @@
+def reverse(number):
+    n_list = list(str(number))
+    number = ""
+    for i in range(1, len(n_list) + 1):
+        number += n_list[-i]
+
+    return int(number)
+
+
+def is_palindrome(number):
+    n_to_check = reverse(number)
+
+    return n_to_check == int(number)   
+
+
+def main():
+    number = eval(input("Enter a number: "))
+    if is_palindrome(number) == True:
```

Comment:
`is_palindrome` returns a boolean, so it is redundant you check this is `True` remember that the block inside an `if` statement will execute when the value is `True`. Do you see my point?